lk2nd/device/dts/msm8916/msm8916-samsung.dts

```diff
@@ -263,6 +260,7 @@
 		compatible = "samsung,grandprimelte", "samsung,gprimeltecan";
 		lk2nd,match-bootloader = "G530FZ*";
 
+		/* FIXME: Use msm8916-samsung-grandprimelte */
 		lk2nd,dtb-files = "msm8916-samsung-gprimeltecan";
```

Comment:
This prop is a priority list so we can do this I think:

```suggestion
		lk2nd,dtb-files = "msm8916-samsung-grandprimelte",
				  "msm8916-samsung-gprimeltecan";
```

so lk2nd will first try lte then can

Reply:
I would like to split it into another issue.